app-main/public/css/style_404.css

```diff
@@ -0,0 +1,40 @@
+body {
+  margin: 0;
+  padding: 0;
+  font-family: system-ui, -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
+    Oxygen, Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue;
+  letter-spacing: 0.02em;
+  background-color: #339989;
+}
+
+#message {
+  background: white;
+  max-width: 360px;
+  border-radius: 3px;
+  font-size: 28px;
+  margin: 100px auto 16px;
+  padding: 32px 24px 16px;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.12), 0 1px 2px rgba(0, 0, 0, 0.24);
+}
+
+#message h2 {
+  color: white;
+  font-weight: bold;
+  font-size: 28px;
+  margin: -32px -24px 8px;
```

Comment:
There's quite a few hard-coded values here which isn't wrong (all of the project styling needs some refactoring and consistency introduced) but in the future you might want to consider something like flexbox which is a useful for layouts and element positioning (it's great for 1 dimensional layouts whereas CSS grid is better for 2D layouts).